vllm/model_executor/layers/activation.py

```diff
@@ -51,17 +55,31 @@ class ScaledActivation(nn.Module):
     def __init__(
         self,
         act_module: nn.Module,
-        hidden_size: int,
+        intermediate_size: int,
         params_dtype: torch.dtype,
```

Comment:
Maybe we can add an option here indicating whether the input is parallelized or not?

Reply:
Oh then what should we do if `input_is_parallel=False`?

Reply:
Got ya. fixed.